### PR TITLE
chore: adjust etcd max_fails for the admin api and add comments

### PR DESCRIPTION
**Pull request by tzssangglass**
### Description

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by tokers — DISMISSED** [review]

File: apisix/core/config_etcd.lua [hunk 1]
```diff
@@ -523,6 +523,7 @@ local function _automatic_fetch(premature, self)
     end
 
     if not (health_check.conf and health_check.conf.shm_name) then
+        -- used for worker processes to synchronously fetch the configuration
```
Comment (spacewander) [inline apisix/core/config_etcd.lua:526]:
```suggestion
        -- used for worker processes to synchronize the configuration
```

**Review by spacewander — APPROVED** [review]

File: apisix/core/config_etcd.lua (hunk 1 above)
Comment (tzssangglass) [reply apisix/core/config_etcd.lua]: updated

File: apisix/core/config_etcd.lua [hunk 2]
```diff
@@ -523,6 +523,7 @@ local function _automatic_fetch(premature, self)
     end
 
     if not (health_check.conf and health_check.conf.shm_name) then
+        -- used for worker processes to synchronize fetch the configuration
```
Comment (spacewander) [inline apisix/core/config_etcd.lua:526]: `synchronize` is already a verb. We usually say `synchronize configuration` but not `synchronously fetch configuration`.

Comment (tzssangglass) [reply apisix/core/config_etcd.lua]: updated

**Review by spacewander — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**